public/css/main.css

```diff
@@ -1,218 +1,238 @@
 /*fork me on GitHub*/
+
 #forkongithub a {
-    background: #e95420;
-    color: #fff;
-    text-decoration: none;
-    font-family: arial, sans-serif;
-    text-align: center;
-    font-weight: bold;
-    padding: 5px 40px;
-    font-size: 1rem;
-    line-height: 2rem;
-    position: relative;
-    transition: 0.5s;
+  background: #e95420;
+  color: #fff;
+  font-family: arial, sans-serif;
+  font-size: 1rem;
+  font-weight: bold;
+  line-height: 2rem;
+  padding: 5px 40px;
+  position: relative;
+  text-align: center;
+  text-decoration: none;
+  transition: .5s;
 }
+
 #forkongithub a:hover {
-    background: orange;
-    color: #fff;
+  background: orange;
+  color: #fff;
 }
-#forkongithub a::before, #forkongithub a::after {
-    content: "";
-    width: 100%;
-    display: block;
-    position: absolute;
-    top: 1px;
-    left: 0;
-    height: 1px;
-    background: #fff;
+
+#forkongithub a::before,
+#forkongithub a::after {
+  background: #fff;
+  content: "";
+  display: block;
+  height: 1px;
+  left: 0;
+  position: absolute;
+  top: 1px;
+  width: 100%;
 }
+
 #forkongithub a::after {
-    bottom: 1px;
-    top: auto;
+  bottom: 1px;
+  top: auto;
 }
 @media screen and (min-width:1333px) {
-    #forkongithub {
-        position: absolute;
-        display: block;
-        top: 0;
-        left: 0;
-        width: 200px;
-        overflow: hidden;
-        height: 200px;
-        z-index: 0;
-    }
-    #forkongithub a {
-        width: 200px;
-        position: absolute;
-        top: 30px;
-        left: -45px;
-        transform: rotate(315deg);
-        -webkit-transform: rotate(315deg);
-        -ms-transform: rotate(315deg);
-        -moz-transform: rotate(315deg);
-        -o-transform: rotate(315deg);
-        box-shadow: 4px 4px 10px rgba(0, 0, 0, 0.8);
-    }
+
+  #forkongithub {
+    display: block;
+    height: 200px;
+    left: 0;
+    overflow: hidden;
+    position: absolute;
+    top: 0;
+    width: 200px;
+    z-index: 0;
+  }
+
+  #forkongithub a {
+    box-shadow: 4px 4px 10px rgba(0, 0, 0, .8);
+    left: -45px;
+    position: absolute;
+    top: 30px;
+    -webkit-transform: rotate(315deg);
```

Comment:
This code reminds me that we should probably try something like autofixer in the future.

Reply:
Agreed. I think with the website redesign I'm probably going to start from scratch on the styling using SCSS, so I can ensure that everything gets autoprefixed as part of the build process :)